t/utime.t

```diff
@@ -8,6 +8,7 @@ use autodie;
 use File::Temp qw(tempfile);
 
 my ($fh, $filename) = tempfile;
+close $fh if $^O eq 'VMS'; # can't utime an open file
```

Comment:
Thanks @craigberry, wouldn't we be better to ALWAYS do this, not just for VMS? Then if it is needed, at least we'll notice the bug before VMS has to suffer with it?

Reply:
I'm ok with always doing it -- I was just trying to limit the scope to where I knew it was needed.

Reply:
yes please!

Reply:
I've revised the commit to close the filehandle without checking platform.  I couldn't build and test it locally since there's no Makefile.PL.